### PR TITLE
Workflowv2 crontriggers list

### PR DESCRIPTION
**Pull request by garbage-collector**
For #1199 

https://github.com/openstack/mistral/blob/master/mistral/api/controllers/v2/cron_trigger.py#L106

### CODE REVIEW

**Comment by garbage-collector** [comment]
Need rebase, waiting workflowv2-workflows-* branches to be accepted before rebasing it.

**Comment by garbage-collector** [comment]
Ready to review

File: openstack/workflow/v2/crontriggers/doc.go [hunk 1]
```diff
@@ -17,25 +17,44 @@ Example to list cron triggers
...
 
-Example to create a cron trigger
+Create a cron trigger. This example will start the workflow "echo" each day at 8am, and it will end after 10 executions.
 
 	createOpts := &crontriggers.CreateOpts{
-		WorkflowID:     "w1",
+		Name: 				"daily",
+		Pattern: 			"0 8 * * *",
+		WorkflowName:     	"echo",
+		RemainingExecutions: 10,
```
Comment (jtopjian) [inline openstack/workflow/v2/crontriggers/doc.go]: nit: Looks like the formatting here is a little off.

File: openstack/workflow/v2/crontriggers/requests.go [hunk 2]
```diff
@@ -78,3 +79,54 @@ func Get(client *gophercloud.ServiceClient, id string) (r GetResult) {
...
 	return
 }
+
+// ListOptsBuilder allows extension to add additional parameters to the List request.
+type ListOptsBuilder interface {
+	ToCronTriggerListQuery() (string, error)
+}
+
+// ListOpts filters the result returned by the List() function.
+type ListOpts struct {
```
Comment (jtopjian) [inline openstack/workflow/v2/crontriggers/requests.go]: I think there are some fields missing here?

https://github.com/openstack/mistral/blob/master/mistral/api/controllers/v2/cron_trigger.py#L106-L112

In addition, I think `SortKey` should be `SortKeys` and `SortDir` should be `SortDirs`?

File: openstack/workflow/v2/crontriggers/doc.go [hunk 3]
```diff
@@ -4,38 +4,70 @@ Package crontriggers provides interaction with the cron triggers API in the Open
...
 
-Example to create a cron trigger
+Create a cron trigger. This example will start the workflow "echo" each day at 8am, and it will end after 10 executions.
 
 	createOpts := &crontriggers.CreateOpts{
-		WorkflowID:     "w1",
+		Name:					"daily",
+		Pattern:				"0 8 * * *",
+		WorkflowName:			"echo",
+		RemainingExecutions:	10,
```
Comment (jtopjian) [inline openstack/workflow/v2/crontriggers/doc.go]: Nit: the formatting of the above lines is still a little off. This is non-blocking, though.

**Pull request merged**